package/Shaders/DeferredCompositeCS.hlsl

```diff
@@ -161,7 +161,7 @@ Texture2D<half4> SpecularSSGITexture : register(t10);
 
 #endif
 
-	MainRW[dispatchID.xy] = min(color, 128);  // Vanilla bloom fix
+	MainRW[dispatchID.xy] = min(color, 250);  // Vanilla bloom fix
```

Comment:
Why this number?

Reply:
The DICE tonemapper and typical ENB tonemappers set the white point as high as this.